test-forge-blocks.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -e
+
+BLOCK=$(cast block-number)
```

Comment:
I'm wondering if we could just combine both scripts in one and just have two workflows. The regular one will run from latest to latest block.